### DISCUSSION TITLE
How can i use time.sleep?

### DISCUSSION THREAD

**Discussion by heav33n** [TomSchimansky/CustomTkinter#1455] [General]
How can i use time.sleep for example to print a text in the window, wait some time with time.sleep and quit app.
Normally it freezes all the app.

**Comment by Akascape** [comment]
@rV4nxZ 
Use `root.after(time_in_millisec, func)`.

Example:
`root.after(5000, root.destroy)`

**Reply by heav33n** [reply]
thanks for replying, it worked perfectly, thank you.
